apps/dsb-client-gateway-api/src/app/modules/message/service/offline-messages.service.ts

```diff
@@ -71,6 +70,21 @@ export class OfflineMessagesService {
         );
       }
 
+      const replyMessages =
+        await this.receivedMessageRepositoryWrapper.repository
```

Comment:
Reply messages must be based on Sent Messages. We want to retrieve how many replies a user sent in response to the message they received.